ui/src/transaction.ts

```diff
@@ -417,6 +417,8 @@ export const unknownOrg: Org = {
 // Identity counter part of `unknownOrg`.
 export const unknownIdentity: Identity = {
   id: "user@unknown.git",
+  peerId: "",
```

Comment:
Shouldn't `transaction.ts` be gone by now? Since we dropped wallet related functionality from mainline.

Reply:
Is it time to delete it? I have no objections to that. It might also necessitate deleting or updating all of the UI components that use `transaction.ts` though.

Reply:
cc @NunoAlexandre because he's working on funding